source/installing.rst

```diff
@@ -19,17 +19,27 @@ Like many other |percona| products, |pdp| is installed by using the
    <https://www.percona.com/doc/percona-repo-config/percona-release.html#percona-release-update-latest-version>`_
    of |percona-release| for details.
 
-As soon as |percona-release| is up-to-date, *set up* the required version of the
-|pdp| product (``ppg-<version>``). For example, to set up |pdp| 11.7, use the following command:
+As soon as |percona-release| is up-to-date, *set up* the
+|pdp| product (``ppg11`). 
 
 .. code-block:: bash
 
-   $ sudo percona-release setup ppg-11.7
+   $ sudo percona-release setup ppg-11
+
+This repository includes the latest version of |pdp|. 
 
 Make sure to run |percona-release| as root or via |sudo|. For the
 sake of convenience, all commands that require elevated privileges start with
 |sudo| in the following procedures.
 
+.. hint::
+
+   To install a specific version of |pdp|, set up the ``ppg-<version>`` product. For example, to set up |pdp| 11.7, use the following command:
```

Comment:
maybe 11.8?

Reply:
I've left 11.7 to show  installing a specific version as some previous version.